test/integration/create_update_delete_test.go

```diff
@@ -12,6 +12,7 @@ import (
 
 	kubernetesfake "k8s.io/client-go/kubernetes/fake"
 
+	"golang.org/x/net/context"
```

Comment:
Why isn't this coming from just "context"?

Reply:
The PR: https://github.com/openshift/cluster-api-provider-aws/pull/118/file